stdlib/curses/__init__.pyi

```diff
@@ -25,3 +26,13 @@ def wrapper(func: Callable[Concatenate[window, _P], _T], /, *arg: _P.args, **kwd
 # it was mapped to the name 'window' in 3.8.
 # Kept here as a legacy alias in case any third-party code is relying on it.
 _CursesWindow = window
+
+# At runtime this class is unexposed and calls itself curses.ncurses_version.
+# That name would conflict with the actual curses.ncurses_version, which is
+# an instance of this class.
+@final
+@type_check_only
+class _ncurses_version(structseq[int], tuple[int, int, int]):
```

Comment:
It also has `__match_args__` on py310+:

```suggestion
class _ncurses_version(structseq[int], tuple[int, int, int]):
    if sys.version_info >= (3, 10):
        __match_args__: Final = ("major", "minor", "patch")
```

(needs `import sys` and `from typing import Final` to be added at the top)

Reply:
huh, stubtest didn't flag that one for me

Reply:
I think because mypy synthesizes `__match_args__` automatically for `NamedTuple` classes, but not for `structseq` classes (which aren't really a "typing feature" in the same way as `NamedTuple` classes). So on `main`, it really isn't missing as an attribute; but with the (correct) change in the base class you're making, it becomes missing.

Reply:
ah that makes sense. It also didn't show up when I re-ran stubtest after the change because I forgot to update my mapping file to reflect that I moved it from `_curses` to `curses`.